### PR TITLE
 Make debowerify work with more than 14 entries

### PR DESCRIPTION
**Pull request by mkoryak**
after 14 entries or so, next entries will overwrite code and cause syntax errors, see #67

this fixes that.

### CODE REVIEW

**Comment by maxnordlund** [comment]
I'm a afraid to merge anymore, @eugeneware could you take a look?

**Comment by mkoryak** [comment]
shouldnt be too afraid,  master is broken right now

**Comment by mkoryak** [comment]
i needed to do the nested arrays thing because you cant safely splice into that array without also moving the computed node boundaries - which we cant do. its easier to add the extra paths as nested array and flatten them later.

**Comment by mkoryak** [comment]
I am testing this on a big project and i found a bug :)

File: index.js [hunk 1]
```diff
@@ -136,14 +136,12 @@ module.exports = function (file, options) {
...
       for (var i = node.range[0] + 1; i < node.range[1]; i++) {
-          chunks[i] = '';
+        chunks[i] = '';
       }
-      paths.forEach(function (p, i) {
-        var st = '\nrequire(' + p + ')'
-        chunks[node.range[1] + (i + 1)] = st
+      chunks[node.range[1] + 1] = paths.map(function (p, i) {
+        return '\nrequire(' + p + ');'
       })
```
Comment (bobthecow) [inline index.js]: Why not just do the `join('\n')` right here?

Comment (mkoryak) [inline index.js]: you are right :+1: fixed in new PR #71

**Pull request closed without merging**